site/content/api/Command-And-Control-API.md

```diff
@@ -3,140 +3,106 @@ title = "Command & Control API"
+| *message-id*     | yes       | *properties*             | *string*     | An identifier that uniquely identifies the message at the sender side. |
 
-# Operations
+The command message MAY contain arbitrary payload to be sent to the device in a single AMQP *Data* section. The value of the message's *subject* property may provide a hint to the device regarding the format, encoding and semantics of the payload data.
 
-The following API defines operations that can be used by *Business Applications* to send commands (which get a response 
-in return) or notification commands to devices over an AMQP 1.0 Network.
+Hono indicates the outcome of the operation by means of the following AMQP delivery states:
 
-## Send a Notification Command
+| Delivery State | Description                                      |
+| :------------- | :----------------------------------------------- |
+| `accepted`    | The command has been delivered to the device for processing. |
+| `released`    | The command has not been delivered to the device because the device is (currently) not connected. |
+| `rejected`    | The command has not been delivered to the device because it does not contain all required information. |
 
-Business Applications use this operation to send a notification command to a device.
+**Note** that Hono relies on the particular protocol adapter to deliver commands to devices. Depending on the used transport protocol and the adapter's implementation, the `accepted` outcome may thus indicate any of the following:
 
-A notification command cannot be responded by the device so the result of this operation is purely based on the outcome of sending
-it to the AMQP 1.0 network (see [Outcomes]({{< relref "#outcomes-for-sending-a-command-or-notification-command">}})).
- 
-Note that the outcome is usually determined by the receiving protocol adapter, which is omitted here for simplicity.
+- An *attempt* has been made to deliver the command to the device. However, it is unclear if the device has received (or processed the command).
+- The device has acknowledged the reception of the command but has not processed the command yet.
+- The device has received and processed the command.
 
-To send a notification command to a device, the first precondition needs to be fulfilled.
+**Examples**
 
-The following sequence diagram shows the message flow with a successful outcome:
+The following sequence diagram shows the successful delivery of a command to a device:
 
 ![Send Notification Command successful outcome](../command_control_notification_Success.png)
 
-The following sequence diagram shows the message flow with an unsuccessful outcome:
+The following sequence diagram shows how the delivery of a command fails because the device is not connected:
```

Comment:
Later the third case is illustrated (delivery of a command failing since the command is malformed), so all cases are covered.
But IMHO it would be even clearer to have this third case covered here already, and then later only refer to the two failing deliveries (since they do not differ for both types of commands).
WDYT?

Reply:
sure, I can move the rejected case here as well.